src/sentry/migrations/0024_auto_20191230_2052.py

```diff
@@ -69,7 +81,7 @@ def _attach_related(_events):
     print("Events to process: {}\n".format(count))
 
     processed = 0
-    for e in RangeQuerySetWrapper(events, step=100, callbacks=(_attach_related,)):
+    for e in RangeQuerySetWrapper(events, step=100, callbacks=(_attach_related, _attach_timestamp,)):
```

Comment:
Any reason for not bundling this into `_attach_related`?

Reply:
I've updated this per your suggestion so we avoid yet another loop through the events.